dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -564,6 +571,9 @@
             <SolidColorBrush x:Key="SystemFillColorCriticalBackgroundBrush" Color="{ThemeResource SystemColorWindowColor}" />
             <SolidColorBrush x:Key="SystemFillColorNeutralBackgroundBrush" Color="{ThemeResource SystemColorWindowColor}" />
 
+            <SolidColorBrush x:Key="SystemControlFocusVisualPrimaryBrush" Color="{StaticResource SystemColorButtonTextColor}"/>
```

Comment:
While you're here, @chingucoding, would you mind updating this to SystemColorWindowTextColor? (We have another issue tracking that request)
```suggestion
            <SolidColorBrush x:Key="SystemControlFocusVisualPrimaryBrush" Color="{StaticResource SystemColorWindowTextColor }"/>
```

Reply:
Sure! Only Primary brush @YuliKl ?